RELEASES.md

```diff
@@ -1,5 +1,11 @@
 # Releases
 
+## 4.0.1 (Sep 24st, 2023)
+
+### Changelog
+
+- #29 [expo] correct export path to support android expo 
```

Comment:
We should also update to v3.0.3 of Application Insights, so that we keep people moving forward.